build.sbt

```diff
@@ -22,7 +22,7 @@ lazy val commonSettings = Seq(
   homepage := Some(url("http://github.com/rklaehn/abc")),
 
   // release stuff
-  credentials += Credentials(Path.userHome / ".ivy2" / ".credentials"),
+  // credentials += Credentials(Path.userHome / ".ivy2" / ".credentials"),
```

Comment:
I think this needs to stay in so I can publish it.

Reply:
Oh, you're right, I forgot about uncommenting it.